rclpy/rclpy/parameter_service.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from rcl_interfaces.srv import DescribeParameters, GetParameters, GetParameterTypes
+from rcl_interfaces.srv import ListParameters, SetParameters, SetParametersAtomically
+from rclpy.parameter import Parameter, PARAMETER_SEPARATOR_STRING
+from rclpy.qos import qos_profile_parameters
+
+
+class ParameterService:
+
+    def __init__(self, node):
+        self._node = node
+        nodename = node.get_name()
+
+        describe_parameters_service_name = '/'.join((nodename, 'describe_parameters'))
```

Comment:
Same as for the parameter separator, it would be better to not redefine this hardcoded separator in every service name definition. Though it looks like that's what's done [currently in rclcpp](https://github.com/ros2/rclcpp/blob/6b34bcc94c12e2213b6de7af57c3d10109402906/rclcpp/src/rclcpp/parameter_service.cpp#L36) so it's not a blocker for getting this merged

Reply:
Constantizing this is straightforward and we've got to start it somewhere. What's a good name for the constant? `TOPIC_SEPARATOR_STRING`?

Reply:
That sounds good to me. It's consistent with the other constant defined in this PR and the naming used in otherp laces like logging https://github.com/ros2/rcutils/blob/4cbd05d9c09886754f41747afa705f15d02c6818/include/rcutils/logging.h#L34